Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1942,17 +1949,20 @@ def renameInputTagsInSequence(self,sequence,oldT="rawDataCollector",newT="rawDat
         self.additionalCommands.append('massSearchReplaceAnyInputTag(process.%s,"%s","%s",False,True)'%(sequence,oldT,newT))
 
     #change the process name used to address HLT results in any sequence
-    def renameHLTprocessInSequence(self,sequence,proc=None,HLTprocess='HLT'):
-        proc = self._options.hltProcess if self._options.hltProcess else self.process.name_()
+    def renameHLTprocessInSequence(self, sequence, proc=None, HLTprocess='HLT', verbosityLevel=1):
+        if proc == None:
+            proc = self._options.hltProcess if self._options.hltProcess else self.process.name_()
         if proc == HLTprocess:
             return
         # look up all module in sequence
-        print("replacing %s process name - sequence %s will use '%s'" % (HLTprocess, sequence, proc))
-        getattr(self.process,sequence).visit(ConfigBuilder.MassSearchReplaceProcessNameVisitor(HLTprocess,proc,whitelist = ("subSystemFolder",)))
+        if verbosityLevel > 0:
+            print("replacing %s process name - sequence %s will use '%s'" % (HLTprocess, sequence, proc))
+        verboseVisit = (verbosityLevel > 1)
+        getattr(self.process,sequence).visit(ConfigBuilder.MassSearchReplaceProcessNameVisitor(HLTprocess, proc, whitelist = ("subSystemFolder",), verbose = verboseVisit))
         if 'from Configuration.Applications.ConfigBuilder import ConfigBuilder' not in self.additionalCommands:
             self.additionalCommands.append('from Configuration.Applications.ConfigBuilder import ConfigBuilder')
-        self.additionalCommands.append('process.%s.visit(ConfigBuilder.MassSearchReplaceProcessNameVisitor("%s", "%s", whitelist = ("subSystemFolder",)))'% (sequence,HLTprocess, proc))
-
+        self.additionalCommands.append(
+          'process.%s.visit(ConfigBuilder.MassSearchReplaceProcessNameVisitor("%s", "%s", whitelist = ("subSystemFolder",), verbose = %s))'% (sequence, HLTprocess, proc, verboseVisit))
```

Comment:
This adds a bit of text to (some of) the cfgs produced by `cmsDriver.py`. Specifically, it adds a `, verbose = False` in the reHLT-related customisations inside the cfg.